caterva2/client.py

```diff
@@ -515,6 +515,42 @@ def copy(self, dst):
         """
         return self.client.copy(self.path, dst)
 
+    def concat(self, srcs, dst, axis):
```

Comment:
I am not sure we want the brief version for `concatenate` name here.  I'd lean towards using the Array API conventions for APIs in general, but shorter versions for the Prompt in the web GUI.  Agreed?